### PR TITLE
Adding Spanish, fixing typo in German

### PR DESCRIPTION
**Pull request by ZuseZ4**
I didn't check the German further, just noticed this typo when I wanted to see how others translated one of the questions.

r? @Kobzol

### CODE REVIEW

File: surveys/2024-annual-survey/translations/es.md [hunk 1]
```diff
@@ -56,42 +56,42 @@ Type: select one
...
 
 Type: select all that apply
 
 - Libros ("The Rust Programming Language", "Rust for Rustaceans", etc.)
-- Ejercicios online (Rustlings, Rust by Example, etc.)
-- Videos
-- 
-- 
-- 
+- Ejercicios online (Rustlings, 100 Excercises To Learn Rust, etc.)
```
Comment (apiraino) [inline surveys/2024-annual-survey/translations/es.md:80]:
```suggestion
- Ejercicios online (Rustlings, 100 Exercises To Learn Rust, etc.)
```

File: surveys/2024-annual-survey/translations/es.md [hunk 2]
```diff
@@ -535,22 +535,22 @@ Type: select all that apply
 - Ciencia de datos
 - Implementación de base de datos
 - Aplicaciones de escritorio
-- Servicios para computadores de escritorio o teléfonos móviles
+- Computadores de escritorio o librerías de teléfonos móviles o servicios
 - Sistemas distribuídos
 - Dispositivos embebidos (con un sistema operativo)
-- Dispositivos embebidos (sin sistema operativo)
+- Dispositivos embebidos (metal puro)
```
Comment (apiraino) [inline surveys/2024-annual-survey/translations/es.md:541]: this was left as "bare metal" in a option of a previous question. Which one is more commonly used?

File: surveys/2024-annual-survey/translations/es.md [hunk 3]
```diff
@@ -579,8 +579,8 @@ Type: matrix
 
 Rows:
 
-- Es fácil para los solicitantes cualificados encontrar trabajos que usan Rust para la mayoría de la programación
-- 
+- Es fácil para los solicitantes cualificados encontrar trabajos que usan Rust para la mayor parte de la programación
+- Es sencillo, para programadores mayores, encontrar empleos en los cuales Rust es utilizado en la mayor parte de la programación.
```
Comment (apiraino) [inline surveys/2024-annual-survey/translations/es.md:583]: programadores or developers? :-) Makes sense using always the same word?

**Review by apiraino — APPROVED** [review]
@ZuseZ4  Aside some small questions, looks good :+1: 

Thanks!

File: surveys/2024-annual-survey/translations/es.md (hunk 3 above)
Comment (ZuseZ4) [reply surveys/2024-annual-survey/translations/es.md:583]: We were using developers everywhere except here, but I changed it now to use programadores.
If we already offer translations, we might as well use spanish words ^^

**Pull request merged**